server/repository/migrations/postgres/2022-04-27T13-00_create_document_table/up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE document (
+    id TEXT NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL,
+    parents TEXT NOT NULL,
+    author TEXT NOT NULL,
+    timestamp TIMESTAMP NOT NULL,
```

Comment:
ok think timestamp fits better here since its the time of the document version. created_datetime sounds like creation time of the first version of the document. Maybe just `datetime`?

Reply:
> created_datetime sounds like creation time of the first version of the document

never mind, timestamp works